css/style.css

```diff
@@ -0,0 +1,248 @@
+body{
+    font-family: 'Poppins', sans-serif;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.4;
+    color: var(--color-white);
+    margin: 0;
+    display: block;
+
+}
+
+.container{
+    
+    background-color: #262647;
+    margin-top: 3px;
+    margin-bottom: 3px;
+    margin-right: 300px;
+    margin-left: 300px;
+    padding: 16px;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+:root{
+    --color-white: #f3f3f3;
+    --color-darkblue: #1b1b32;
+    --color-darkblue-alpha: rgba(27,27,50,0.8);
+    --color-green: #37af65;
+}
+
+body::before{
+    content: '';
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    z-index: -1;
+    background: var(--color-darkblue);
+    background-image: linear-gradient(115deg, rgba(58,58,158,0.8), rgba(136,136,206,0.7)), 
+    url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+*::before{
+    box-sizing: border-box;
+}
+
+.header{
+    padding: 0 0.625rem;
+    margin-bottom: 1.875rem;
+    display: block;
+}
+
+.text-center{
+    text-align: center;
+    
+}
+
+h1{
+    margin-top: 0;
+    margin-bottom: 0.5rem;
+    font-weight: 400;
+    line-height: 1.5rem;
+    display: block;
+    font-size: 2em;
+    margin-block-start: 0.67em;
+    margin-block-end: 00.67em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+}
+
+.discripton{
+    font-style: italic;
+    font-weight: 200;
+    text-shadow: 1px 1px 1px rgb(0, 0,0 / 40%);
+
+
+}
+
+div{
+    display: block;
+}
+
+.form{
+    padding: 2.5rem;
+    background: var(--color-darkblue-alpha);
+    border-radius: 0.25rem;
+    display: block;
+    margin-top: 0em;
+    align-content: center;
+}
+
+.form-group{
+    margin: auto 1.25rem auto;
+    padding: 0.25rem;
+}
+
+label{
+    display: flex;
+    align-items: center;
+    font-size: 1.125rem;
+    margin-bottom: 0.5rem;
+    cursor: default;
+    align-content: center;
+}
+
+.form-control{
+    display: inline-block;
+    width: 100%;
+    height: 2.375rem;
+    padding: 0.375rem 0.75rem;
+    color: #fff;
+    background-clip: padding-box;
+    border: 1px solid #ced4da;
+    border-radius: 00.25rem;
+    transition: border-color 0.15s ease-out, box-shadow 0.15s ease-in-out ;
+}
+
+input{
+    margin: 0;
+    font-family: inherit;
+    font-size: inherit;
+    line-height: inherit;
+    writing-mode: horizontal-tb !important;
+    
+}
+
+select{
+    margin: 0;
+    font-size: inherit;
+    font-family: inherit;
+    line-height: inherit;
+    writing-mode: horizontal-tb !important;
+
+}
+
+select:not(:-internal-list-box){
+    overflow: visible !important;
```

Comment:
Try as much as you can to avoid using too many of the !important in your CSS as this is not a best practice in css

Reply:
- [ ] **[OPTIONAL]:** Please consider being more specific in your explanations, "Is not a best practice" is too vague of a reason! Why `!important` should be avoided as much as possible in your styles? It would be great if you change this! 🤓 🛠️